packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.controller.js

```diff
@@ -262,6 +264,64 @@ export default class MoveResumeCtrl {
         assign(moveData, {
           meeting,
         });
+
+        // Set address complements for copper line creation if set
+        if (this.offer.selected.searchAddress.residence) {
+          assign(moveData, {
+            residence: this.offer.selected.searchAddress.residence,
+          });
```

Comment:
```suggestion
          Object.assign(moveData, {
            residence: this.offer.selected.searchAddress.residence,
          });
```
maybe we don't need lodash in this particular case